app/config/services/admin.xml

```diff
@@ -585,6 +585,14 @@
             <argument />
         </service>
 
+        <service id="app.admin.mooc_quizz" class="AppBundle\Admin\MoocQuizzAdmin">
+            <tag name="sonata.admin" manager-type="orm" label="Quizzs" group="Mooc" />
```

Comment:
`Quizz` is invariable since it finishes with a `z` (epic french rules ftw \o/)
so it's `Quizz` even for plural